test/cli/knexfile-test.spec.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const path = require('path');
+const { FileTestHelper, execCommand } = require('cli-testlab');
+
+const KNEX = path.normalize(__dirname + '/../../bin/cli.js');
+
+describe('knexfile resolution', () => {
+  /**
+   * @type FileTestHelper
+   */
+  let fileHelper;
+  beforeEach(() => {
+    fileHelper = new FileTestHelper(path.resolve(__dirname, '../jake-util'));
+    fileHelper.deleteFile('test.sqlite3');
```

Comment:
this is to prevent preexisting file from messing up tests. could be moved to before block, but that would require some juggling around fileHelper instantiation